device/workstation/w_linux.go

```diff
@@ -116,9 +116,10 @@ func (l *linux) Unmount() error {
 }
 
 const diskSelectionTries = 3
+const writeAttempts = 3
 
 // Notifies user to chose a mount, after that it tries to write the data with `diskSelectionTries` number of retries
-func (l *linux) WriteToDisk(img string) (progress chan bool, err error) {
+func (l *linux) WriteToDisk(img string) (job *help.BackgroundJob, err error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 16 of function (*linux).WriteToDisk() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->